src/stores/RoomViewStore.tsx

```diff
@@ -93,17 +91,16 @@ type Listener = (isActive: boolean) => void;
  * A class for storing application state for RoomView.
  */
 export class RoomViewStore extends EventEmitter {
-    // Important: This cannot be a dynamic getter (lazily-constructed instance) because
-    // otherwise we'll miss view_room dispatches during startup, breaking relaunches of
-    // the app. We need to eagerly create the instance.
-    public static readonly instance = new RoomViewStore(defaultDispatcher);
-
-    private state = INITIAL_STATE; // initialize state
+    // initialize state as a copy of the initial state. We need to copy else one RVS can talk to
+    // another RVS via INITIAL_STATE as they share the same underlying object. Mostly relevant for tests.
+    private state = JSON.parse(JSON.stringify(INITIAL_STATE));
```

Comment:
Shouldn't a shallow clone be enough? If not, I'd suggest using lodash's `cloneDeep` instead

Reply:
Or our own `utils.deepCopy` for consistency with the codebase

Reply:
Needs some actioning